Website/htdocs/mpmanager/packages/inensus/ticket/src/app/Models/OutsourceReport.php

```diff
@@ -17,5 +17,5 @@
  */
 class OutsourceReport extends Model
 {
-    protected $table = 'ticket_outsource_reports';
+    protected $table = 'outsource_reports';
```

Comment:
please check the ticket table migration. The table name should `ticket_outsource_reports`